src/sql/src/session/vars.rs

```diff
@@ -1205,6 +1205,13 @@ pub const ENABLE_SESSION_RBAC_CHECKS: ServerVar<bool> = ServerVar {
     internal: false,
 };
 
+pub const ENABLE_PER_REPLICA_NOTICE: ServerVar<bool> = ServerVar {
```

Comment:
I think you could add an entry to the `feature_flags` macro call like this:
```
(
	enable_per_replica_notice,
	"Whether to print a notice when querying per-replica introspection sources.",
	/* default */ true,
	/* internal */ false
)
```

The formatting of the help message might be off, but nothing we can't fix.

Reply:
I wanted this to be a session var, not a server var, as I don't think it makes sense for it to be turned off globally rather than per-user. `feature_flags!` only supports server vars.